android/daemon/tunnel/libwg-go/jni.c

```diff
@@ -23,8 +23,14 @@ extern char* wgVersion();
 JNIEXPORT jint JNICALL Java_org_mozilla_firefox_vpn_daemon_VPNService_wgTurnOn(
     JNIEnv* env, jclass c, jstring ifname, jint tun_fd, jstring settings) {
   const char* ifname_str = (*env)->GetStringUTFChars(env, ifname, 0);
+  if (ifname_str == NULL) {
+    return -1;
+  }
   size_t ifname_len = (*env)->GetStringUTFLength(env, ifname);
   const char* settings_str = (*env)->GetStringUTFChars(env, settings, 0);
+  if (settings_str == NULL) {
+    return -1;
+  }
```

Comment:
I might be exposing my ignorance with this question, but what is this change about?

Reply:
Nevermind, I read the description of the PR and I see it's an unrelated nullptr check :)

Reply:
My current suspicion is that connection health might pass something wierd to wg-go that makes it panic, as we know wg-go did never panic so far. So maybe i messed up something and we passed a "null" ref :)